packages/odyssey-react/src/components/TextInput/TextInput.tsx

```diff
@@ -143,47 +179,61 @@ export const TextInput = withTheme(
           }
         : {};
 
-    const input = (
-      <input
-        {...omitProps}
-        {...ariaProps}
-        className={styles.root}
-        disabled={disabled}
-        id={oid}
-        name={name}
-        onChange={handleChange}
-        onBlur={onBlur}
-        onFocus={onFocus}
-        placeholder={placeholder}
-        readOnly={readonly}
-        ref={ref}
-        required={required}
-        type={type}
-        defaultValue={defaultValue}
-        value={value}
-      />
+    const prefixStyles = useCx(
+      styles.prefix,
+      isSearchTextInput && styles.affixIcon
```

Comment:
Not particular to this review - but this useCx hook is non-obvious on what it does and why a boolean is passed as the second argument. What does `Cx` even mean?